TabloidMVC/Startup.cs

```diff
@@ -27,7 +27,11 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<ICategoryRepository, CategoryRepository>();
             services.AddTransient<IPostRepository, PostRepository>();
             services.AddTransient<IUserProfileRepository, UserProfileRepository>();
```

Comment:
Could not test due to build errors due to leftover merge conflict markers in Startup.cs